.github/workflows/executor.yml

```diff
@@ -1,16 +1,39 @@
-name: Executor Deploy
+name: Executor
 
 on:
+  pull_request:
+    paths:
+      - "client/packages/executor/**/*.ts"
+      - "client/packages/executor/**/*.json"
   push:
     branches:
       - development
 
-env:
-  RUST_BACKTRACE: 1
-
 jobs:
-  build:
+  test:
+    name: Test Executor
+    defaults:
+      run:
+        working-directory: client/packages/executor
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - run: cd .. && make
+      - run: yarn
+      - run: npm ci
+      - run: npm test
```

Comment:
Soon we will be able to test everything as a workspace with the stuff I have done in private 🥳